components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/util/SAMLSSOUtil.java

```diff
@@ -2547,4 +2547,24 @@ public static boolean isSignatureValid(AuthnRequest authnRequest, String querySt
             return false;
         }
     }
+
```

Comment:
We need method level comment, explaining what this method supposed to do, what it expects as parameters, and what it returns.

Reply:
Added method level comment.